src/fragment-components/a-inline-loading.tsx

```diff
@@ -0,0 +1,251 @@
+import React from 'react';
+import {
+	RadioButtonGroup,
+	RadioButton,
+	InlineLoading,
+	TextInput,
+	NumberInput
+} from 'carbon-components-react';
+import { AComponent } from './a-component';
+import { ComponentInfo } from '.';
+import { css } from 'emotion';
+import image from './../assets/component-icons/inline-loading.svg';
+import {
+	nameStringToVariableString,
+	reactClassNamesFromComponentObj,
+	angularClassNamesFromComponentObj
+} from '../utils/fragment-tools';
+
+export const AInlineLoadingSettingsUI = ({ selectedComponent, setComponent }: any) => {
+	return <>
+		<legend className='bx--label'>Status</legend>
+		<RadioButtonGroup
+		orientation='vertical'
+		name='status-radio-buttons'
+		valueSelected={selectedComponent.status}
+		onChange={(event: any) => setComponent({
+			...selectedComponent,
+			status: event
+		})}>
+			<RadioButton
+				id='inactive'
+				labelText='Inactive'
+				value='inactive' />
+			<RadioButton
+				id='active'
+				labelText='Active'
+				value='active' />
+			<RadioButton
+				id='finished'
+				labelText='Finished'
+				value='finished' />
+			<RadioButton
+				id='error'
+				labelText='Error'
+				value='error' />
+		</RadioButtonGroup>
+		<TextInput
+			value={selectedComponent.activeText}
+			labelText='Active text'
+			onChange={(event: any) => setComponent({
+				...selectedComponent,
+				activeText: event.currentTarget.value
+			})} />
+		<TextInput
+			value={selectedComponent.inactiveText}
+			labelText='Inactive text'
+			onChange={(event: any) => setComponent({
+				...selectedComponent,
+				inactiveText: event.currentTarget.value
+			})} />
+		<TextInput
+			value={selectedComponent.errorText}
+			labelText='Error text'
+			onChange={(event: any) => setComponent({
+				...selectedComponent,
+				errorText: event.currentTarget.value
+			})} />
+		<TextInput
+			value={selectedComponent.successText}
+			labelText='Success text'
+			onChange={(event: any) => setComponent({
+				...selectedComponent,
+				successText: event.currentTarget.value
+			})} />
+		<TextInput
```

Comment:
Put icon descriptions in an "Accessibility" accordion so they don't take that much space on the screen.

They default to the accompanying text anyway and might not even be used.

Put it at the end of the section (after all other settings)